src/pages/QuestionPage.tsx

```diff
@@ -63,6 +64,11 @@ export default function QuestionPage() {
         {gotQuestion && prompt && answers && index ?
           <VStack>
             <Logo size={["32px", "50px"]} />
+            {isHost ?
```

Comment:
If you don't want anything there if `isHost` is `false`, then you should use this notation:

```suggestion
            {isHost &&
```